yb-voyager/cmd/root.go

```diff
@@ -52,6 +52,9 @@ Refer to docs (https://docs.yugabyte.com/preview/migrate/) for more details like
 			if cmd.Parent() != nil && cmd.Parent().Use != "yb-voyager" {
 				cmdName = fmt.Sprintf("%s-%s", cmd.Parent().Use, cmd.Use)
 			}
+			if cmd.Parent().Use == "data" { //import data file case
+				cmdName = fmt.Sprintf("%s-%s-%s", cmd.Parent().Parent().Use, cmd.Parent().Use, cmd.Use)
+			}
```

Comment:
See if we can make it more readable by using `cmd.CommandPath()` (need to trim the leading `yb-voyager` and replace space with a hyphen)